.github/workflows/release.yml

```diff
@@ -5,12 +5,34 @@ on:
     types: [prereleased, released]
 
 jobs:
+  build:
+    name: Build integration for multiple platforms
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [ linux ]
+        goarch: [ amd64, arm64 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build integration
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+        run: |
+          make compile
+      - name: Upload artifact for docker build step
+        uses: actions/upload-artifact@v2
+        with:
+          name: nri-kubernetes-${{ matrix.goos }}-${{ matrix.goarch }}
+          path: bin/nri-kubernetes-${{ matrix.goos }}-${{ matrix.goarch }}
+
   release:
     name: Release docker images and config
     runs-on: ubuntu-latest
     env:
-      DOCKER_IMAGE_NAME: newrelic/infrastructure-k8s
+      DOCKER_IMAGE_NAME: roobre/nri-kubernetes # TODO: Change this back
```

Comment:
is this intentional?

Reply:
Yes, just to avoid pushing `-pre` images to prod. It will be reverted once we test it working.